hw/ip/edn/doc/theory_of_operation.md

```diff
@@ -28,19 +30,25 @@ There are two broad modes for state machine control: auto request mode and boot-
 Random values are needed by peripherals almost immediately after reset, so to simplify interactions with the boot ROM, boot-time request mode is the default mode.
 
 In boot-time request mode, the command sequence is fully hardware-controlled and no command customization is possible.
-In this mode, the EDN automatically issues a special reduced-latency `instantiate` command followed by the default `generate` commands.
+In this mode, EDN automatically issues a special reduced-latency `instantiate` command followed by the default `generate` commands.
```

Comment:
I think the rest of this document uses "the EDN". That makes sense if you read it as "the Entropy Distribution Network", and we should probably keep this consistent.

Reply:
I agree, that's done now. Thanks!